src/bowlingGame.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+class BowlingGame {
+  constructor() {
+    this.rolls = [];
+  }
+
```

Comment:
may be worth thinking about storing the frames as there own class, this will help separate things so that you don't just have a long string of numbers to deal with.